kasa/smart/smartmodule.py

```diff
@@ -138,7 +138,9 @@ def query(self) -> dict:
 
         Default implementation uses the raw query getter w/o parameters.
         """
-        return {self.QUERY_GETTER_NAME: None}
+        if self.QUERY_GETTER_NAME:
+            return {self.QUERY_GETTER_NAME: None}
+        return {}
```

Comment:
We want to have this covered as it's part of the "core" api.

Reply:
Test added